readme.md

```diff
@@ -25,8 +25,16 @@ jss source.css -p > source.jss
 ### API
 
 ```javascript
-import {cssToJss} from 'jss-cli'
-cssToJss(css, options)
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
+var jss = css2jss(css)
+console.log(JSON.stringify(jss, null, 2))
```

Comment:
just that i need to add the babelify transform then - because es6 is not well supported yet and i'd like to use it in the browser. I can "es6ify" it again, sure that that is a good thing?

I always think - people who know es6, know how to rewrite, but those who don't need to learn es6 + jss, which is "one more thing".